README.md

```diff
@@ -35,7 +35,7 @@ then compile:
 $ ninja -C build/release
 ```
 
-If you're missing a dependency of Seastar, then it is possible to have the configuration process fetch a version of the dependency locally for development.
+If you're missing the dependency of Seastar, then it is possible to have the configuration process fetch a version of the dependency locally for development.
```

Comment:
This is a wrong correction... Seastar has many (some might say too many...) dependencies, and you may be missing any one of them. So the article "a" is the right one. "some" would have also worked. "the" is wrong.